indico/modules/events/editing/settings.py

```diff
@@ -14,7 +14,9 @@
 
 
 editing_settings = EventSettingsProxy('editing', {
-    'review_conditions': OrderedDict()
+    'review_conditions': OrderedDict(),
+    'editable_types': OrderedDict({'paper': True, 'slides': False, 'poster': False}),
```

Comment:
I don't think we need an OrderedDict here - especially since you lose the ordering by initializing it from a regular dict!

Reply:
If the OrderedDict is initialized properly here why shouldn't we keep it?

Reply:
what's the benefit in maintaining an ordered structure for something's that just a collection of "enabled/disabled" flags?

Reply:
So that they appear in the same order in all events, and they don't arbitrarily change when saving the editable types.

Reply:
I'd prefer to have the order hardcoded on the client side... using the the list of enabled/disabled types for this feels very hacky.

Reply:
Isn't hardcoding it on the client side hacky as well? What do the others think: @indico/development-team

Reply:
The display order sounds like something very much client-related to me...

Reply:
Yeah, first of all, you should initialize the `OrderedDict` with a list of items to preserve the order. However, I think that the way wee store that data shouldn't matter that much and we can easily sort it on the client side.